ludwig/schema/metadata/trainer_metadata.py

```diff
@@ -361,18 +374,6 @@
                                                          expected_impact=ExpectedImpact.MEDIUM,
                                                          literature_references=None,
                                                          internal_only=False),
-     'increase_batch_size_on_plateau_max': ParameterMetadata(ui_display_name='Batch Size Increase On Plateau: Cap',
```

Comment:
@geoffreyangus we should also add a backwards compatibility hook as we do for other removed params so we rename `increase_batch_size_on_plateau_max` to `max_batch_size` when we detect it in an old config.

Reply:
ah okay, I was wondering about this. I'll take a look there. Thanks!

Reply:
Done.